helm/common/templates/_healthchecks.tpl

```diff
@@ -16,8 +16,8 @@ Daemon startup probe settings
 startupProbe:
   tcpSocket:
     port: external-port
-  failureThreshold: {{ .healthcheck.startup.failureThreshold }}
-  periodSeconds: {{ .healthcheck.startup.periodSeconds }}
+  failureThreshold: {{ default .Values.healthcheck.startup.failureThreshold 30 }}
+  periodSeconds: {{ default .Values.healthcheck.startup.periodSeconds 10 }}
```

Comment:
and here

Reply:
and same as above

Reply:
Also intentional and could go either way. I think the defaults here are just an added security but not necessry if defaults are stored within at least some `values.yml (whether that includes from a dependency library or application chart).